docs/ota-client-guide/modules/ROOT/pages/upload-large-binary.adoc

```diff
@@ -28,17 +28,17 @@ garage-sign init \
 ----
 +
 
-. Pull the latest version of the `targets.json` file to your local repository.
+. Pull the latest version of the `targets.json` file.
 +
 [source,bash]
 ----
 garage-sign targets pull \
   --repo <localreponame>
 ----
 
-. Upload your binary file to your local repository and specify the name and version of your `targets.json` file.
+. Upload your binary to OTA Connect.

```

Comment:
You aren't specifying the name and version of the `targets.json` file, you're specifying the name and version of the target you're adding to the `targets.json` file--i.e. the "software version". (Note: this is a place where our terminology choice makes for some awkward sentences. "Specify the name and version of your software version" isn't a good look.)

```suggestion
. Upload your binary to OTA Connect. You'll need to specify the name and version of the binary you're uploading.

```